src/NServiceBus.RavenDB/SessionManagement/RavenDbStorageSession.cs

```diff
@@ -10,11 +10,6 @@
 
     class RavenDbStorageSession : Feature
     {
-        public RavenDbStorageSession()
-        {
-            DependsOnAtLeastOne(typeof(RavenDbSagaStorage), typeof(RavenDbOutboxStorage));
-        }
```

Comment:
Should there still be a constructor that enables by default? Or why is removing the the dependency information altogether the best strategy?